### ISSUE TITLE
Lint/DisjunctiveAssignmentInConstructor errors with NoMethodError against empty constructors

### ISSUE THREAD

**Issue by rmm5t** [rubocop-hq/rubocop#6678]
The `Lint/DisjunctiveAssignmentInConstructor` cop errors when if finds an empty constructor.

**I am working on a branch to fix this. I should have a PR submitted shortly.**

--------

## Expected behavior

`Lint/DisjunctiveAssignmentInConstructor` cop to pass given an empty constructor:

```
def initialize
end
```

## Actual behavior

Rubocop errors out with a `NoMethodError`

```
An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspected exmample.rb:2

NoMethodError:
       undefined method `type' for nil:NilClass
```

## RuboCop version

```
$ bundle exec rubocop -V
0.63.0 (using Parser 2.6.0.0, running on ruby 2.5.3 x86_64-darwin17)
```

(but I think this was actually introduced in 0.62.x)

**Comment by bkuhlmann** [comment]
I'm afraid this is only partially fixed. I discovered you can produce the error with the following code snippet (i.e. test.rb): 

``` ruby
# frozen_string_literal: true

class NullLogger
  def warn _message
  end
end
```

This yields the following output:

``` bash
cop test.rb

An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspecting /Users/bkuhlmann/Dropbox/Development/OSS/test.rb:4:2.
To see the complete backtrace run rubocop -d.
Inspecting 1 file
An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspecting /Users/bkuhlmann/Dropbox/Development/OSS/test.rb:4:2.
To see the complete backtrace run rubocop -d.
.

1 file inspected, no offenses detected

1 error occurred:
An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspecting test.rb:4:2.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
https://github.com/rubocop-hq/rubocop/issues

Mention the following information in the issue report:
0.63.0 (using Parser 2.6.0.0, running on ruby 2.6.0 x86_64-darwin18)
```

This make using the null object pattern show up with errors. :cry:

**Comment by bkuhlmann** [comment]
I was able to isolate the above error further. It happens only with `#initialize` constructor. Example:

``` ruby
class NullLogger
  def initialize _parameters
  end
end
```

**Comment by rmm5t** [comment]
@bkuhlmann This fix hasn't been released yet. i.e. It's only in the `master` branch. My guess is that the fix will be released in a future v0.63.1.

> `Mention the following information in the issue report:`
> `0.63.0`

Did you test this against the v0.63.0 release or against a build from the `master` branch?

**Comment by rmm5t** [comment]
For more about the release schedule, see the Changelog:
https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md

**Comment by koic** [comment]
FYI, RuboCop 0.63.1 has been released.

**Comment by bkuhlmann** [comment]
Thanks! Picked up the latest update and all is green again. :)
